include/cudaJIT.h

```diff
@@ -1,4 +1,6 @@
 // NVRTC Compiler for JITing GPU functions
+#pragma once
```

Comment:
what is this pragma for?

Reply:
It's a header include guard. It's not standard but is widely supported by the compilers we use (at least gcc and clang) and is shorter than the whole `#ifndef INCLUDE_CUDA_JIT #define INCLUDE_CUDA_JIT #endif // INCLUDE_CUDA_JIT` song and dance